src/components/UserPortal/ChatRoom/ChatRoom.tsx

```
@@ -30,10 +30,8 @@
 
 import React, { useEffect, useRef, useState, useCallback } from 'react';
 import type { ChangeEvent } from 'react';
-import SendIcon from '@mui/icons-material/Send';
-import { Button, Dropdown, Form, InputGroup } from 'react-bootstrap';
+import { Button } from 'react-bootstrap';
```

ESLint

[error] 'Button' import from 'react-bootstrap' is restricted. Direct imports of Button from react-bootstrap are not allowed. Use the shared Button component from src/shared-components/Button/ instead.

(no-restricted-imports)